test/plug/mailbox_preview_test.exs

```diff
@@ -110,6 +116,26 @@ defmodule Plug.Swoosh.MailboxPreviewTest do
     end
   end
 
+  describe "/" do
+    test "with existing messages redirects to most recent" do
+      opts = MailboxPreview.init(storage_driver: StorageDriver)
+
+      conn = conn(:get, "/")
+      conn = MailboxPreview.call(conn, opts)
+
+      assert conn.state == :sent
+      assert conn.status == 302
+      assert get_resp_header(conn, "location") == ["/1"]
+    end
+
+    test "with no messages redirects to base path" do
+      opts = MailboxPreview.init(storage_driver: EmptyDriver)
+      conn = conn(:get, "/latest")
```

Comment:
Still using "latest" here. I would fix it myself but I'm on my phone :( need to run formatter to pass ci.